COPYRIGHT.txt

```diff
@@ -110,6 +110,11 @@ Copyright: 2007, Starbreeze Studios
  2014-2019, Godot Engine contributors.
 License: Expat and Zlib
 
+Files: ./thirdpart/assimp/
+Comment: Open Asset Import (Assimp)
+Copyright: 2006-2016, assimp team
+License: BSD-2-clause
```

Comment:
It's BSD-**3**-clause: https://github.com/assimp/assimp/blob/master/LICENSE

Reply:
Updated.